admin-dev/themes/new-theme/js/product-page/product-header.js

```diff
@@ -41,8 +41,6 @@ export default function() {
     $navWidth += $(item).width();
   });
 
-  $navTabs.width($navWidth);
```

Comment:
@YeLnatSs 😮I'm surprised this does the job. Can you tell us more about why `$navTabs.width($navWidth);` was creating a bug ? What is the value `$navWidth` for polish BO ?

Reply:
@matks It gave the element a fixed width so when I was changing for combination the last element was no longer visible. 
The issue was for polish BO but I tried with others languages.

Reply:
Maybe rather than not calling this function, we need to call it again when you change for combination ?

Without this call, this whole JS function looks useless 🤔and I dont think the people who wrote it did it for nothing: it solves some visual issues I think. So I'm afraid if we remove it, we'll introduce some old issues.

Reply:
No prob, I'll look for something else

Reply:
It's not removed, it's only called later with a wider width `$navTabs.width($widthWithTabs);`
This actually seems logic because `$widthWithTabs` was not used otherwise

Reply:
My bad then 😉

Reply:
No it's better 😛 initially it was not called later 😉 https://github.com/PrestaShop/PrestaShop/pull/13660/commits/59c12d3eb9ee1821e086baef7d11ad5ee33fc045